src/NHapi.Base/Parser/XMLParser.cs

```diff
@@ -830,6 +821,10 @@ public override XmlDocument EncodeDocument(IMessage source)
                 return null;
             }
 
+            public override void Parse(IMessage message, string @string)

```

Comment:
is this requried? might be worth adding comments as to why it has no behaviour?

Reply:
It was only added to satisfy `ParserBase` abstraction, and `AnonymousClassXMLParser` is only used in the test harness method (which could probably be removed in the future since we don't use them.